.github/workflows/build.yml

```diff
@@ -40,12 +46,12 @@ jobs:
         run: docker compose --profile gh-actions up -d
 
       - name: Run gradle build
-        run: ./gradlew build --parallel
+        run: ${gradle_cmd} build --parallel
```

Comment:
```suggestion
        run: ${gradle_cmd} build
```

Reply:
I have merged this pr, so I would add these changes in my open pr with Redis